packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageList.tests.js

```diff
@@ -816,7 +879,27 @@ QUnit.module('MessageList', moduleConfig, () => {
             });
         });
 
-        QUnit.test('should be scrolled to last message if items changed at runtime with an invalidate call', function(assert) {
+        QUnit.test('should be scrolled down if typingUsers changed at runtime', function(assert) {
```

Comment:
This looks unexpectedly to me

Reply:
https://github.com/DevExpress/DevExtreme/pull/28229#discussion_r1818731799